src/orka-builder.ts

```diff
@@ -70,7 +70,9 @@ export default class OrkaBuilder {
     );
     this.useCors();
     this.use(() => addRequestId(this.config));
-    this.use(() => addVisitorId(this.config));
+    if (this.config.visitor && this.config.visitor.orkaEnabled) {
```

Comment:
🙃 I prefer enabled instead of orkaEnabled

Reply:
I though of writing this as `enabled` but I think that will be misleading for the apps that use orka. Do you want to call it just `orka`?

Reply:
ditto